Add mode for "Closure Stylesheets (GSS)"

dbarnett opened the pull request:
The only difference from standard CSS so far is that it parses syntax properly inside `@component` blocks and also doesn't indent them.

Fixes #3470.

marijnh commented:
Merged as e79d4ffafb3300 . Though I'm somewhat confused why you went to the effort of adding a test without making sure it actually passed. Fixed that in 5d853959ca8a

dbarnett commented:
Weird. I did some last-minute refactors, but thought I verified tests pass after those. Which ones failed? (Travis results look broken for a different reason.)

marijnh commented:
For one thing, the `getMode` call didn't actually name the mode correctly.

dbarnett commented:
Yeah, I moved the test file and changed the name and at least thought I re-ran tests. No idea about the other change preventing negative indent, though. Didn't see any failures at all and I think that one would have shown up before my last-minute changes.

Anyway, sorry for test failures. I'll do better next time. =)

dbarnett commented:
How often do you do version bumps? It looks like there hasn't been a new version since 5.6.1, before this change.

marijnh commented:
Once per month. 5.7 is coming out early next week.
